src/audiences/ui/components/rule.js

```diff
@@ -221,15 +224,26 @@ export default function Rule( props ) {
 				>
 					{ __( 'Select a field', 'altis-analytics' ) }
 				</option>
-
-				{ fields.map( fieldData => (
-					<option
-						key={ fieldData.name }
-						value={ fieldData.name }
-					>
-						{ fieldData.label }
-					</option>
-				) ) }
+				<optgroup label={ __( 'Persistent Data', 'altis-analytics' ) }>
+					{ persistentFields.map( fieldData => (
+						<option
+							key={ fieldData.name }
+							value={ fieldData.name }
+						>
+							{ fieldData.label }
+						</option>
+					) ) }
+				</optgroup>
+				<optgroup label={ __( 'Point in time data', 'altis-analytics' ) }>
```

Comment:
`Data` should be consistently capitalised here; also added some hyphens for fun

```suggestion
				<optgroup label={ __( 'Point-in-time Data', 'altis-analytics' ) }>
```